src/huntsman/pocs/observatory.py

```diff
@@ -310,66 +320,79 @@ def take_dark_fields(self,
         if not isinstance(n_darks, list):
             n_darks = listify(n_darks) * len(exptimes)
 
+        if set_from_config:
+            dark_config = self.get_config('calibration', default=dict())
```

Comment:
I think the dark config should be `self.get_config('calibs.dark', default=dict())`. Then we can set the flat fields up consistently so their config lives in `self.get_config('calibs.flats'). Then you wouldn't have to use the `dark` key in the lines below, which is a bit cleaner.